helper.php

```diff
@@ -220,7 +220,7 @@ public function ensureAbsoluteId($id) {
      * @param Doku_Renderer_xhtml $R
```

Comment:
I mean in the files `helper.php`, `syntax/list.php` etc. 
For example: `Doku_Renderer_xhtml` at line 220, can now be updated to `Doku_Renderer`
Further the before_val, before_item,after_...(), etc doesn't have PHPDocs